ashpd-demo/src/portals/desktop/location.rs

```diff
@@ -63,6 +63,7 @@ mod imp {
     }
     impl ObjectImpl for LocationPage {
         fn constructed(&self, _obj: &Self::Type) {
+            // TODO Localize strings.
```

Comment:
Don't for now, as we use from_str to re-get the original enums back

Reply:
This is not valid anymore.